packages/manager/apps/container/src/container/common/utils.ts

```diff
@@ -0,0 +1,54 @@
+import { useReket } from '@ovh-ux/ovh-reket';
+
+export type PciProject = {
+  access: string;
+  creationDate: Date;
+  description: string;
+  expiration: Date;
+  manualQuota: boolean;
+  orderId: unknown;
+  planCode: string;
+  projectName: string;
+  project_id: string;
```

Comment:
maybe use camelCase also on projectId ?

Reply:
@y4nnL , it's due to an API constraint ?

Reply:
This type is basically here to describe the API response so the compiler does not complain. If camel case is mandatory we could cast the response even though it's not ideal. You tell me

Reply:
Ok thanks for the explanation :)